src/tokens/spacings.js

```diff
@@ -0,0 +1,14 @@
+const unit = 8;
+
+const spacings = {
+  xxs: `${unit / 4}px`,
+  xs: `${unit / 2}px`,
+  s: `${unit}px`,
+  m: `${(3 * unit) / 2}px`,
+  l: `${unit * 2}px`,
+  xl: `${(5 * unit) / 2}px`,
+  xxl: `${unit * 3}px`,
+  xxxl: `${unit * 4}px`,
```

Comment:
Let's keep these as absolute values. I remember I had only asked you to do it based on computations but it's very hard to read and becomes complicated. Let's go for simplicity